internal/host/transition_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/filanov/stateswitch"
```

Comment:
May be opinionated, but shouldn't we own that?

Reply:
That's up to the stateswitch BDFL @filanov

Reply:
i don't mind to move it